ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Common/screen.cpp

```diff
@@ -31,10 +31,17 @@ namespace enigma {
 unsigned int bound_framebuffer = 0; //Shows the bound framebuffer, so glGetIntegerv(GL_FRAMEBUFFER_BINDING_EXT, &fbo); don't need to be called (they are very slow)
 
 void graphics_set_viewport(float x, float y, float width, float height) {
-  //NOTE: OpenGL viewports are bottom left unlike Direct3D viewports which are top left
+  //NOTE: OpenGL viewports are bottom left unlike Direct3D viewports which are top left 		
   y = enigma_user::window_get_height() - y - height;
   glViewport(x,y,width,height);
   glScissor(x,y,width,height);
+  
+}
+
+void graphics_remove_garbage(float x,float y, float width,float height){
```

Comment:
I again recommend making this just a `graphics_enable_scissor_test` rather than the more specific `graphics_remove_garbage`—chances are, down the line, someone else will wish they could disable the scissor test, too.

I thought I remembered implementing some wrappers around these already, actually. There should be enable functions for fog and stippling, etc. Maybe put it with those?